tests/test.linter.js

```diff
@@ -899,21 +899,24 @@ describe('Linter.extractMetadata()', function() {
     var addonLinter = new Linter({
       _: ['tests/fixtures/angular-unadvised-library.xpi'],
```

Comment:
As mentioned on irc the fixture data can be deleted if it's not used any where else.